README.md

```diff
@@ -14,70 +14,173 @@ Takeaway Challenge
+jonnyabrams@JONNYs-MacBook-Pro takeaway-challenge % irb -r './lib/order'
+3.0.2 :001 > order = Order.new
+ => 
+#<Order:0x000000013a0f01f0
+... 
+jonnyabrams@JONNYs-MacBook-Pro takeaway-challenge % irb -r './lib/order'
+3.0.2 :001 > order = Order.new
+ => 
+#<Order:0x000000013f84fe78
+... 
+3.0.2 :002 > order.select_dish('carbonara')
+ => 1 
+3.0.2 :003 > order.select_dish('arrabiata', 3)
+ => 3 
+3.0.2 :004 > order.remove_dish('carbonara')
+ => 1 
+3.0.2 :005 > order.remove_dish('arrabiata', 2)
+ => 2 
+3.0.2 :006 > order.select_dish('bolognese')
+ => 1 
+3.0.2 :007 > order.basket
+ => {"carbonara"=>0.0, "arrabiata"=>4.5, "bolognese"=>7.5} 
+3.0.2 :008 > order.calculate_total
+ => 12.0 
+```
 
-In code review we'll be hoping to see:
+I also added a 'fail' guard clause in case of unavailable dishes:
 
-* All tests passing
-* High [Test coverage](https://github.com/makersacademy/course/blob/main/pills/test_coverage.md) (>95% is good)
-* The code is elegant: every class has a clear responsibility, methods are short etc.
+```
+jonnyabrams@JONNYs-MacBook-Pro takeaway-challenge % irb -r './lib/order'
+3.0.2 :001 > order = Order.new
+ => 
+#<Order:0x0000000149a940d0
+... 
+3.0.2 :002 > order.select_dish('gnocchi')
+/Users/jonnyabrams/Projects/Makers/takeaway/takeaway-challenge/lib/order.rb:14:in `select_dish': This is not on the menu (RuntimeError)
+        from (irb):2:in `<main>'
+        from /Users/jonnyabrams/.rvm/rubies/ruby-3.0.2/lib/ruby/gems/3.0.0/gems/irb-1.3.5/exe/irb:11:in `<top (required)>'
+        from /Users/jonnyabrams/.rvm/rubies/ruby-3.0.2/bin/irb:23:in `load'
+        from /Users/jonnyabrams/.rvm/rubies/ruby-3.0.2/bin/irb:23:in `<main>'
+```
+
+Then it was time to implement a checkout method sending a confirmation text via the Twilio API:
+
+```
+jonnyabrams@JONNYs-MacBook-Pro takeaway-challenge % irb -r './lib/order'
+3.0.2 :001 > order = Order.new
+ => 
+#<Order:0x00000001450f58c0
+... 
+3.0.2 :002 > order.select_dish('carbonara', 3)
+ => 3 
+3.0.2 :003 > order.select_dish('bolognese')
+ => 1 
+3.0.2 :004 > order.select_dish('puttanesca', 2)
+ => 2 
+3.0.2 :005 > order.checkout
+carbonara: 19.5
+bolognese: 7.5
+puttanesca: 11.0
+*************
+Your total is 38.0
+ => <Twilio.Api.V2010.MessageInstance body: Sent from your Twilio trial account - Thank you! Your order was placed and will arrive by 11:50 num_segments: 1 direction: outbound-api from: +16626678674 to: +447867513304 date_updated: 2022-03-06 10:51:00 +0000 price:  error_message:  uri: /2010-04-01/Accounts/ACadafd609098a3ba714c6f6e1917a6ce4/Messages/SM3bd3c82ea0f54256898ed9e9b6e47948.json account_sid: ACadafd609098a3ba714c6f6e1917a6ce4 num_media: 0 status: queued messaging_service_sid:  sid: SM3bd3c82ea0f54256898ed9e9b6e47948 date_sent:  date_created: 2022-03-06 10:51:00 +0000 error_code:  price_unit: USD api_version: 2010-04-01 subresource_uris: {"media"=>"/2010-04-01/Accounts/ACadafd609098a3ba714c6f6e1917a6ce4/Messages/SM3bd3c82ea0f54256898ed9e9b6e47948/Media.json"}> 
```

Comment:
This is very interesting, I don't think I was able to directly check that a message had been sent correctly - did you manage to use this return value for your tests?